src/Components/Helper/Action/Action.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
We should not put those values in a `.json` file. Let use javascript file

- [ ] refactor